python/test/unit/language/test_core.py

```diff
@@ -3852,6 +3852,9 @@ def matmul_kernel(
 @pytest.mark.parametrize("in_type_str", ['float8e5', 'float8e4nv'])
 @pytest.mark.parametrize("low_precision_acc", [0, 32, 64, 128])
 def test_fp8_dot_acc(in_type_str, low_precision_acc, device):
+    if is_hip():
+        pytest.skip('test_fp8_dot_acc not supported on HIP.')
```

Comment:
is this because some fp8 types are not supported? If some maybe this should go in `check_type_supported`?